lib/amorail/entities/company.rb

```diff
@@ -1,49 +1,20 @@
 module Amorail
   class AmoCompany < Amorail::AmoEntity
-    set_amo_name "company"
-    set_amo_response_name "contacts"
-    
-    attr_accessor :name, :linked_leads_id,
-                  :email, :phone, :address, :website
+    include Leadable
+    amo_names 'company', 'contacts'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.